### DISCUSSION TITLE
Installing as beginner

### DISCUSSION THREAD

**Discussion by 70percentsale** [Dineshkarthik/telegram_media_downloader#132] [Q&A] [answered]
Hey, i am really new to githhub. Anyone can guide start from install to configure the files?

**Comment by Dineshkarthik** [comment, marked as answer]
Hi,
- To install the script and its dependencies follow the [Installation instructions](https://github.com/Dineshkarthik/telegram_media_downloader#installation)
- To get telegram api hash and api id and how to configure your `config.yaml` file follow the [Configuration Instructions](https://github.com/Dineshkarthik/telegram_media_downloader#installation)
- Finally to execute the script and download from telegram follow the [Execution Instructions](https://github.com/Dineshkarthik/telegram_media_downloader#installation)

In short all you need to install and run this script you can find it in the [Readme](https://github.com/Dineshkarthik/telegram_media_downloader#readme)
